module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AccountMenuTest.php

```diff
@@ -0,0 +1,349 @@
+<?php
+/**
+ * Mink account ajax menu test class.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2011.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFindTest\Mink;
+
+/**
+ * Mink account ajax menu test class.
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class AccountMenuTest extends \VuFindTest\Unit\MinkTestCase
+{
+    use \VuFindTest\Unit\UserCreationTrait;
+    use \VuFindTest\Unit\DemoDriverTestTrait;
+
+    /**
+     * Standard setup method.
+     *
+     * @return mixed
+     */
+    public static function setUpBeforeClass()
+    {
+        return static::failIfUsersExist();
+    }
+
+    /**
+     * Standard setup + login
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        // Give up if we're not running in CI:
+        if (!$this->continuousIntegrationRunning()) {
+            return $this->markTestSkipped('Continuous integration not running.');
+        }
+        // Setup config
+        $this->changeConfigs([
+            'Demo' => $this->getDemoIniOverrides(),
+            'config' => [
+                'Catalog' => ['driver' => 'Demo'],
+                'Authentication' => [
+                    'enableAjax' => true,
+                    'enableDropdown' => false
+                ]
+            ]
+        ]);
+    }
+
+    protected function setJSStorage($states)
```

Comment:
Do you mind adding function comments to the undocumented functions in here?

Reply:
Will do